roles/neutron-common/defaults/main.yml

```diff
@@ -53,37 +61,6 @@ neutron:
       - neutron-sriov-nic-agent
       - neutron-usage-audit
```

Comment:
The reason I originally excluded it here and explicitly did the update-alternatives elsewhere it is because we didn't want the lbaas binary to be installed if the flag wasn't set.  I'll go ahead and make the change though.